docs/source/using/managing-minishift.adoc

```diff
@@ -275,3 +275,29 @@ Use the `openshift-version` option to request a specific version of OpenShift. Y
 
 The Minishift VM is exposed to the host system with a host-only IP address that
 can be obtained with the `minishift ip` command.
+
+[[connecting-with-ssh]]
+== Connecting to the Minishift VM with SSH
+
+You can use the xref:../command-ref/minishift_ssh{outfilesuffix}[`minishift ssh`] command to interact with the Minishift VM.
+
+When you run this command, it checks whether you have an external SSH client installed on your machine. An SSH client is not a
+prerequisite, but it impacts the behavior of the command.
+
+If you have an external SSH client::
+You can run `minishift ssh` without a sub-command to open an interactive shell using your external SSH client.
```

Comment:
Are you sure about this? In one of the other comments you mentioned that Linux/Mac come with an SSH client by default, so from that statement we can conclude that having an SSH client is actually more common. Also, the only reason that we needed to work around no SSH client in Windows is because of a bug in OpenSSH, which means that in the very near future (after this bug is fixed) we might even change this text to recommend/require an SSH client.